telepathy-sasl-signon/empathy-uoa-auth-handler.c

```diff
@@ -403,7 +410,9 @@ empathy_uoa_auth_handler_supports (EmpathyUoaAuthHandler *self,
     TpAccount *account)
 {
   const gchar *provider;
-  EmpathySaslMechanism mech;
+
+  //EmpathySaslMechanism mech;
+  (void)self; // suppress unused parameter warning
```

Comment:
I prefer to use -Wno-unused-parameter to avoid littering the source code with useless statements